frontend/vite.config.ts

```diff
@@ -6,7 +6,7 @@ export default defineConfig({
   plugins: [
     preact(),
     replace({
-      API_URL: JSON.stringify('wss://api.scrum-poker.aws.tngtech.com'),
+      API_URL: JSON.stringify('ws://localhost:8080'),
```

Comment:
Will this work when we deploy on production? I am not familiar enough with `vite` to be certain about this

Reply:
No, this will cause issues. Maybe we can inject the actual production URL via environment variable? That way, one could still have a production build that runs against the local API if you want to.

Reply:
I added an environment variable `API_URL` that can be used to override the local URL in `npm start`, `npm run preview` and `npm run build`.

Reply:
This will need to become part of the deployment script, though.